Plugins/DotNET/DotNETExports.cpp

```diff
@@ -226,8 +224,25 @@ static void StackPushString(const char* value)
     }
     else
     {
-        LOG_WARNING("Failed to push string '%s' - recursion level %i.",
-            str.m_sString, vm->m_nRecursionLevel);
+        LOG_WARNING("Failed to push string '%s' - recursion level %i.", str.m_sString, vm->m_nRecursionLevel);
+    }
+}
+
+static void StackPushRawString(const char* value)
+{
+    auto vm = Globals::VirtualMachine();
+    ASSERT(vm->m_nRecursionLevel >= 0);
+
+    LOG_DEBUG("Pushing string '%s'.", value);
+    CExoString str(value);
+
+    if (vm->StackPushString(str))
+    {
+      ++s_pushedCount;
```

Comment:
nit, but this change has inconsistent indentation. Would be a lot less of an issue if it wasn't bundled with a massive reformat :P